app/views/spree/taxons/_filters.html.erb

```diff
@@ -24,11 +24,13 @@
         </div>
       </div>
     </div>
-    <div class="col-xl-5 col-lg-7" data-hook="taxon_products">
-      <!-- <div class="sorting-filter d-flex align-items-center justify-content-end"> -->
-        <!-- <label>Sort by:</label> -->
-        <%= render 'sorting_filters', taxon: @taxon %>
```

Comment:
Why we removed this partial? Also the below filters code should be inside this partial. Any specific reason

Reply:
sir i have not done this so these changes are not on me